src/api/app/views/webui2/webui/staging/workflows/_overall_state.html.haml

```diff
@@ -1 +1,9 @@
-= project.name
+%div{ class: "state-#{staging_project.overall_state}" }
+  %div
+    -# TODO: Link to the staging project show page
```

Comment:
:eyes:

Reply:
We need to wait for the staging project show page to be done... so there's not much we can do now...